tests/test_internals/test_tools.py

```diff
@@ -34,3 +46,33 @@ def test_make_nwbfile_from_metadata(self):
             ),
         ):
             make_nwbfile_from_metadata(metadata=dict())
+
+
+@pytest.mark.skipif(
+    not HAVE_GLOBUS or not LOGGED_INTO_GLOBUS,
```

Comment:
Maybe the equivalent:
```suggestion
    not (HAVE_GLOBUS and LOGGED_INTO_GLOBUS)
```
Is more clear?